pkg/apis/crd/v1alpha1/types.go

```diff
@@ -345,6 +345,9 @@ const (
 	// NetworkPolicyConditionRealizable means the condition stores information about
 	// various realizable conditions of the NetworkPolicy.
```

Comment:
Probably we should rephrase this one too:

NetworkPolicyConditionRealizable reports whether the NetworkPolicy is realizable and the reasons why it is not.

Reply:
done.